client/src/components/Forms/signup/index.jsx

```diff
@@ -0,0 +1,171 @@
+/* eslint-disable no-console */
+import React, { useState } from 'react';
+import axios from 'axios';
+import {
+  Modal,
+  Button,
+  Form,
+  Input,
+  message,
+} from 'antd';
+import logo from '../../../assets/images/logo.png';
+import './Signup.css';
+
+function Signup() {
+  const [isModalVisible, setIsModalVisible] = useState(false);
+  const [fullName, setFullName] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [confirm, setConfirm] = useState('');
+  const [rool] = useState('');
+
+  const signUp = async () => {
+    try {
+      await axios.post('/api/signup', {
+        fullName,
+        email,
+        password,
+        rool,
+      });
+      setIsModalVisible(false);
+      message.success('Welcome to Nova');
+    } catch (err) {
+      message.error('Sorry, try again');
+      console.log(err);
+    }
+  };
+
+  const showModal = () => {
+    setIsModalVisible(true);
+  };
+
+  const handleOk = () => {
+    setIsModalVisible(false);
+  };
+
+  return (
+    <>
+      <Button type="primary" onClick={showModal}>
+        Open Modal
+      </Button>
+      <Modal
+        title={(
+          <div className="modal-logo-img">
+            <img src={logo} alt="logo" className="logo-img" />
+          </div>
+        )}
+        visible={isModalVisible}
+        onOk={handleOk}
+        footer={null}
```

Comment:
You will not add the logic of modal visibility here, just add the form and handle its functionality.

Reply:
If I delete it it gives me an error because I am not a user at the same time, I must be in the code
![image](https://user-images.githubusercontent.com/79801554/166068552-6924a06b-84a7-4b50-88d6-d465047e5418.png)